Tests/designspaceLib/split_test.py

```diff
@@ -211,3 +211,10 @@ def test_optional_min_max_internal(condition, expected_set: ConditionSet):
     """Check that split's internal helper functions produce the correct output
     for conditions that are partially unbounded."""
     assert _conditionSetFrom([condition]) == expected_set
+
+
+def test_avar2(datadir):
+    ds = DesignSpaceDocument()
+    ds.read(datadir / "test_avar2.designspace")
+    out = list(splitInterpolable(ds))[0][1]
```

Comment:
this test is a bit cryptic, what's that `[0][1]` indices, what is `out`?  maybe use self-explanatory variables or add comment to avoid one having to go read the source code to unsterstand what's going on

Reply:
it's also not exercising the splitInterpolable method much, as the test designspace contains a single space sub-space/VF setup so you're just asserting that it passes through. Better than nothing of course